lib/public/Comments/ICommentsManager.php

```diff
@@ -276,6 +277,55 @@ public function create($actorType, $actorId, $objectType, $objectId);
 	 */
 	public function delete($id);
 
+	/**
+	 * Get comment related with user reaction
+	 *
+	 * Throws PreConditionNotMetException when the system haven't the minimum requirements to
+	 * use reactions
+	 *
+	 * @param integer $parentId
+	 * @param string $actorType
+	 * @param string $actorId
+	 * @param string $reaction
+	 * @return IComment
+	 * @throws NotFoundException
+	 * @throws PreConditionNotMetException
+	 * @since 24.0.0
+	 */
+	public function getReactionComment(int $parentId, string $actorType, string $actorId, string $reaction): IComment;
+
+	/**
+	 * Retrieve all reactions with specific reaction of a message
+	 *
+	 * @param integer $parentId
+	 * @param string $reaction
+	 * @return IComment[]
+	 * @since 24.0.0
+	 */
+	public function retrieveAllReactionsWithSpecificReaction(int $parentId, string $reaction): ?array;
```

Comment:
```suggestion
	public function retrieveAllReactionsWithSpecificReaction(int $parentId, string $reaction): array;
```

Should be changed on the Manager too. always returns an array. In worst case it's empty.
Null should not be returned